test/functional/feature_assumeutxo.py

```diff
@@ -114,6 +114,13 @@ def expected_error(log_msg="", rpc_details=""):
                 f.write(valid_snapshot_contents[(32 + 8 + offset + len(content)):])
             expected_error(log_msg=f"[snapshot] bad snapshot content hash: expected a4bf3407ccb2cc0145c49ebba8fa91199f8a3903daf0883875941497d2493c27, got {wrong_hash}")
 
+        self.log.info("  - snapshot file with invalid size, too small")
```

Comment:
nit: I think this info log could be improved if you retouch. Testing the file size is not really the goal, it's about hitting the metadata parsing error that was added, right? So I would suggest to call it something like "Inablity to parse metadata fails early" or so.

Reply:
Thanks! I agree, I'll update it if I have to retouch it and if not, could be done in a follow-up.

Reply:
done!